public/scripts/app.js

```diff
@@ -1,7 +1,85 @@
 console.log("Sanity Check: JS is working!");
 
 $(document).ready(function(){
+  console.log("Document has loaded!")
 
-// your code
 
-});
+  let successPostAlert = `<div class="alert alert-success" role="alert">Contact successfully added!</div>`
+  let errorAlert = `<div class="alert alert-danger" role="alert">Oops! Something went wrong!</div>`
+
+  $.ajax({
+    method: 'GET',
+    url: '/api/contacts',
+    success: onSuccess,
+    error: onError
+  });
+
+  $('#addContact').on('click', function(e) {
+    $('#contactModal').modal(); //triggers modal
+    console.log("modal open!")
+      $('form').on('submit', function(e) {
```

Comment:
Adding this event listener as part of the `click` handler is confusing--you should set up this callback once, when the document is ready, not set up the callback every time the button is clicked. (Right now, if I click the button to open the modal, then close the modal, then open it again, this handler will be added twice, and it'll send 2 AJAX requests when the form is eventually submitted.)